### PR TITLE
feat: set theme to scrollbar #1120

### LINKED ISSUE #1120: Stylize Scrollbars to Match Theme

### PR DESCRIPTION
**Pull request by aalencar**
Using colors: `--card8`, `--text0`, and `--text2`

Tested on MacOS.

Frame 1

resolves #1120

### CODE REVIEW

**Comment by aalencar** [comment]
@sandruparo please, send the colors here and I will update later.

**Comment by mturoci** [comment]
@aalencar make sure the CSS works cross browsers (at least Chrome + Firefox + Safari)

**Comment by aalencar** [comment]
@mturoci thanks for reminding. Firefox uses different css props and it was broken. Now it's fixed.

Updated the description with screenshots for the browsers I tested.

**Comment by aalencar** [comment]
@sandruparo and I agreed on `--text2` for scrollbar thumb.

File: ui/src/index.scss [hunk 1]
```diff
@@ -13,6 +13,26 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 */
+* {
```
Comment (mturoci) [inline ui/src/index.scss:16]: Would specifying it on `body` instead of `*` work also?

Comment (aalencar) [reply ui/src/index.scss:16]: No

**Pull request merged**